zerver/tests/test_home.py

```diff
@@ -271,7 +271,7 @@ def test_home(self) -> None:
 
         # Verify succeeds once logged-in
         with (
-            self.assert_database_query_count(53),
+            self.assert_database_query_count(54),
```

Comment:
`anonymous_group_ids` is empty at https://github.com/zulip/zulip/blob/95bc5f85976b229a491ace27ccc167a1b65df022/zerver/lib/streams.py#L1039 before this commit. While now, we have 3 anonymous group ids. Resulting in this extra usergroupmembership query:
```
#38
sql: (SELECT "zerver_usergroupmembership"."user_group_id" AS "col1", "zerver_usergroupmembership"."user_profile_id" AS "col2", 'user' AS "member_type" FROM "zerver_usergroupmembership" WHERE "zerver_usergroupmembership"."user_group_id" IN (33, 34, 35)) UNION (SELECT "zerver_groupgroupmembership"."supergroup_id" AS "col1", "zerver_groupgroupmembership"."subgroup_id" AS "col2", 'group' AS "member_type" FROM "zerver_groupgroupmembership" WHERE "zerver_groupgroupmembership"."supergroup_id" IN (33, 34, 35))
```